### PR TITLE
dev-refactor: update purpose configs

### PR DESCRIPTION
**Pull request by beetlecrunch**

### CODE REVIEW

File: kubernetes/configs/dev-refactor/commons.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@ KAFKA_BROKERS="boot-yqksbq44.c3.kafka-serverless.eu-central-1.amazonaws.com:9098
 AGREEMENT_TOPIC="event-store.dev-refactor_agreement.events"
 CATALOG_TOPIC="event-store.dev-refactor_catalog.events"
 ATTRIBUTE_TOPIC="event-store.dev-refactor_attribute_registry.events"
+PURPOSE_TOPIC="event-store.dev-refactor_purpose.events"
```
Comment (galales) [inline kubernetes/configs/dev-refactor/commons.sh:23]: the purpose topic should be added also to the `notifier-seeder`

File: kubernetes/overlays/purpose-process/dev-refactor/configmap.yaml [hunk 2]
```diff
@@ -4,9 +4,7 @@ metadata:
   name: {{SERVICE_NAME}}
   namespace: {{NAMESPACE}}
 data:
-  STORAGE_TYPE: "{{PURPOSE_PROCESS_STORAGE_TYPE}}"
-  STORAGE_CONTAINER: "{{PURPOSE_PROCESS_STORAGE_CONTAINER}}"
```
Comment (galales) [inline kubernetes/overlays/purpose-process/dev-refactor/configmap.yaml:8]: isn't the process still creating the document on S3?

File: kubernetes/overlays/purpose-process/dev-refactor/deployment.yaml [hunk 3]
```diff
@@ -19,38 +19,28 @@ spec:
                 configMapKeyRef:
                   name: {{SERVICE_NAME}}
                   key: AUTHORIZATION_MANAGEMENT_URL
-            - name: CATALOG_MANAGEMENT_URL
+            - name: CATALOG_PROCESS_URL
               valueFrom:
                 configMapKeyRef:
                   name: {{SERVICE_NAME}}
```
Comment (galales) [inline kubernetes/overlays/purpose-process/dev-refactor/deployment.yaml]: The catalog process is not called by the purpose
```suggestion
```

File: kubernetes/overlays/purpose-process/dev-refactor/deployment.yaml [hunk 4]
```diff
@@ -19,38 +19,24 @@ spec:
...
                   key: AUTHORIZATION_MANAGEMENT_URL
-            - name: CATALOG_MANAGEMENT_URL
+                  key: CATALOG_PROCESS_URL
```
Comment (micdes-pagopa) [inline kubernetes/overlays/purpose-process/dev-refactor/deployment.yaml]: multiple key fields, typo?

**Review by micdes-pagopa — COMMENTED** [review]
Missing `PURPOSE_READMODEL_WRITER_SERVICE_NAME `in `kubernetes/configs/shared.sh`

File: kubernetes/overlays/purpose-process/dev-refactor/deployment.yaml (hunk 4 above)
Comment (beetlecrunch) [reply kubernetes/overlays/purpose-process/dev-refactor/deployment.yaml]: Just a typo, thanks

**Review by micdes-pagopa — DISMISSED** [review]

File: kubernetes/overlays/notifier-seeder/dev-refactor/configmap.yaml [hunk 5]
```diff
@@ -7,4 +7,5 @@ data:
   KAFKA_CLIENT_ID: "{{NAMESPACE}}-pagopa-interop-notifier-seeder"
   KAFKA_GROUP_ID: "{{NAMESPACE}}-notifier-seeder"
   CATALOG_TOPIC: "{{CATALOG_TOPIC}}"
+  PURPOSE_TOPIC: "{{PURPOSE_TOPIC}}"
```
Comment (galales) [inline kubernetes/overlays/notifier-seeder/dev-refactor/configmap.yaml:10]: the variable should be added also to the deployment

**Review by galales — DISMISSED** [review]
one more point to address, approving in advance

**Pull request merged**